appengine/remote/remote-server/src/main/java/com/example/appengine/remote/RemoteServlet.java

```diff
@@ -1,5 +1,5 @@
-/**
- * Copyright 2016 Google Inc.
+/*
+ * Copyright 2015 Google Inc.
```

Comment:
In general (and I saw this earlier), we don't change the dates.  (Rolling them forward or backward).